en-US/firefox/whatsnew_66.lang

```diff
@@ -12,6 +13,10 @@ Get a Firefox Account – Keep your data private, safe and synced
 Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.
 
 
+;Congrats! You’re using the latest version of Firefox.
+Congrats! You’re using the latest version of Firefox.
```

Comment:
Great, another string that was completely missed in l10n copy hand-off
https://github.com/mozilla/bedrock/pull/6812/files

We should import this translation from existing files :-\